_opensearch/rest-api/_script-apis/create-stored-script.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Create or Update Stored Script
+parent: Script APIs
+grand_parent: REST API reference
+nav_order: 1
+---
+
+## Create or update stored script
+
+Creates or updates a stored script or search template.
+
+For additonal information about Painless scripting, see:
+
+* [k-NN Painless Scripting extensions]({{site.url}}{{site.baseurl}}/search-plugins/knn/painless-functions/).
+
+* [k-NN]({{site.url}}{{site.baseurl}}/search-plugins/knn/inswz/).
+
+
+### Path parameters
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| script-id | String | Stored script or search template ID. Must be unique across the cluster. Required. |
+
+### Query parameters
+
+All parameters are optional.
+
+| Parameter | Data Type | Description | 
+:--- | :--- | :---
+| context | String | Context in which the script or search template is to run. To prevent errors, the API immediately compiles the script or template in this context. |
+| cluster_manager_timeout | Time | Amount of time to wait for a connection to the master node. Defaults to 30 seconds. |
```

Comment:
We need to decide what term to use to replace "master node" (in running text). Primary node? We use "primary" and secondary shard.

Reply:
oh, actually the new node name is "cluster_manager" node.

Reply:
yeah primary and secondary are best I think. I'll go ahead and update